sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -1993,8 +1994,7 @@ public List<CaseMergeIndexDto[]> getCasesForDuplicateMerging(
 				cb.lessThanOrEqualTo(root2.get(Case.CREATION_DATE), DateHelper.getStartOfDay(criteria.getCreationDateFrom())),
 				cb.greaterThanOrEqualTo(root2.get(Case.CREATION_DATE), DateHelper.getEndOfDay(criteria.getCreationDateTo()))));
 
-		Predicate filter = CriteriaBuilderHelper
-			.and(cb, createDefaultFilter(cb, root), createDefaultFilter(cb, root2), userFilter, criteriaFilter, nameSimilarityFilter, diseaseFilter);
+		Predicate filter = CriteriaBuilderHelper.and(cb, userFilter, criteriaFilter, criteriaFilter2, nameSimilarityFilter, diseaseFilter);
```

Comment:
Not sure this is needed. It was not asked by the issue you worked on.

Reply:
fixed